### PR TITLE
Feature/index cta

### PR DESCRIPTION
**Pull request by roblim**
- add dynamic cta footer to index page
- add `show_cta_footer` key to config: 

```
data_docs_sites:
  # Data Docs make it simple to visualize data quality in your project. These
  # include Expectations, Validations & Profiles. The are built for all
  # Datasources from JSON artifacts in the local repo including validations &
  # profiles from the uncommitted directory. Read more at https://docs.greatexpectations.io/en/0_8_1+57_g744fdf3b_dirty/features/data_docs.html
  local_site:
    class_name: SiteBuilder
    store_backend:
      class_name: FixedLengthTupleFilesystemStoreBackend
      base_directory: uncommitted/data_docs/local_site/
    site_index_builder:
      show_cta_footer: true
```

- added more descriptive headers to index page
- made breadcrumb area sticky

Screenshot 2019-10-23 14 40 01
Screenshot 2019-10-23 14 40 38

### CODE REVIEW

File: great_expectations/render/renderer/site_builder.py [hunk 1]
```diff
@@ -402,6 +405,17 @@ def add_resource_info_to_index_links_dict(self,
     
         return index_links_dict
 
+    #TODO: Fill out this method to make cta footer dynamic
```
Comment (roblim) [inline great_expectations/render/renderer/site_builder.py:408]: This is the method that generates the cta_object that gets passed to the index page renderer.

File: great_expectations/render/renderer/site_builder.py [hunk 2]
```diff
@@ -410,6 +424,9 @@ def build(self):
         resource_keys = target_store.list_keys()
         index_links_dict = OrderedDict()
 
+        if self.show_cta_footer:
```
Comment (roblim) [inline great_expectations/render/renderer/site_builder.py:427]: `show_cta_footer` comes from new key in config file, which gets passed to index builder when site builder instantiates it

File: great_expectations/render/renderer/site_builder.py [hunk 3]
```diff
@@ -410,6 +424,9 @@ def build(self):
...
+            index_links_dict["cta_object"] = self.get_cta_object()
```
Comment (roblim) [inline great_expectations/render/renderer/site_builder.py:428]: `index_links_dict` has a new `cta_object` key that index page renderer uses to render cta footer if present

File: great_expectations/render/renderer/site_index_page_renderer.py [hunk 4]
```diff
@@ -420,7 +435,12 @@ def render(cls, index_links_dict):
             })
             sections.append(section)
 
-        return RenderedDocumentContent(**{
+        index_page_document = RenderedDocumentContent(**{
                 "utm_medium": "index-page",
                 "sections": sections
             })
+        
+        if cta_object:
```
Comment (roblim) [inline great_expectations/render/renderer/site_index_page_renderer.py:443]: Add cta footer here if the appropriate key was found in the `index_links_dict`.

File: great_expectations/render/view/templates/index_page.j2 [hunk 5]
```diff
@@ -16,28 +16,34 @@
...
-            {% include 'ge_info.j2' %}
+          {% include 'ge_info.j2' %}
         </div>
       </div>
     </div>
-    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.15.0/umd/popper.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+
+    {% if cta_footer %}
```
Comment (roblim) [inline great_expectations/render/view/templates/index_page.j2:44]: Render cta footer if found in the index page document

File: great_expectations/render/view/templates/cta_footer.j2 [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    <span aria-hidden="true">&times;</span>
+  </button>
+</footer>
+<script>
+  if (sessionStorage.getItem("showCta") !== "false") {
+    $('#ge-cta-footer').removeClass("invisible")
+  }
+  $('#ge-cta-footer').on('closed.bs.alert', function () {
+    sessionStorage.setItem("showCta", false)
+  })
```
Comment (roblim) [inline great_expectations/render/view/templates/cta_footer.j2:27]: This is the logic that persists dismissed state of cta footer.

**Review by eugmandel — APPROVED** [review]

**Pull request merged**